TUM Campus App/MenuTableViewCell.swift

```diff
@@ -12,10 +12,37 @@ class MenuTableViewCell: CardTableViewCell {
 
     override func setElement(_ element: DataElement) {
         if let menu = element as? CafeteriaMenu {
-            dishLabel.text = menu.name
-            priceLabel.text = ""
+            
+            //Dish Label
+            dishLabel.text = menu.details.nameWithEmojiWithoutAnnotations
+            
+            //Price Label
+            if let menuPrice = menu.price?.st {
```

Comment:
Before doing that we might need a way to figure out if the user is student, employee or guest